src/Sarif.Converters/FxCopConverter.cs

```diff
@@ -59,16 +59,9 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
 
             if (rules.Count > 0)
             {
-                IDictionary<string, Rule> rulesDictionary = new Dictionary<string, Rule>();
-
-                foreach (Rule rule in rules)
-                {
-                    rulesDictionary[rule.Id] = rule;
-                }
-
                 run.Resources = new Resources
                 {
-                    Rules = rulesDictionary
+                    Rules = rules
```

Comment:
>rules [](start = 28, length = 5)

Well, _that's_ a nice simple change!

Reply:
:)

---
In reply to: [249144017](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249144017) [](ancestors = 249144017)